src/driver/landing_pages.ml

```diff
@@ -0,0 +1,136 @@
+open Packages
+open Odoc_unit
+
+let make_unit ~odoc_dir ~odocl_dir ~mld_dir ~output_dir rel_path ~content
+    ?(include_dirs = []) ~pkgname ~pkg_args () =
+  let input_file = Fpath.(mld_dir // rel_path / "index.mld") in
+  let odoc_file = Fpath.(odoc_dir // rel_path / "page-index.odoc") in
+  let odocl_file = Fpath.(odocl_dir // rel_path / "page-index.odocl") in
+  let () = Util.write_file input_file (String.split_on_char '\n' content) in
+  let parent_id = rel_path |> Odoc.Id.of_fpath in
+  {
+    parent_id;
+    odoc_dir;
+    input_file;
+    output_dir;
+    odoc_file;
+    odocl_file;
+    pkg_args;
+    pkgname;
+    include_dirs;
+    index = None;
+    kind = `Mld;
+  }
+
+module PackageLanding = struct
+  let content pkg =
```

Comment:
I've refactored the `content` functions to take advantage of `Format` in https://github.com/panglesd/odoc/pull/32